daemon/config_windows.go

```diff
@@ -0,0 +1,26 @@
+package daemon
+
+import (
+	"os"
+
+	flag "github.com/docker/docker/pkg/mflag"
+)
+
+// PlatformSpecific defines the configuration of a docker daemon
+// that is specific to a particular platform.
+type PlatformSpecific struct {
```

Comment:
@jhowardmsft nope type embedding doesn't work that way. once you do what I showed above you can just refer to it like `config.TheFieldThatsInCommonConfig` :smile: https://golang.org/doc/effective_go.html#embedding